src/components/sendOTP/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next';
+
+import SignUpHeader from 'src/components/reusable/header';
+import OTPMessageField from 'src/components/sendOTP/OTPMessageField';
+import SuccessfulVerification from 'src/components/sendOTP/SuccessfulVerification';
+
+const AccountVerification = () :JSX.Element => {
+    const { t } = useTranslation();
+    const [isSubmitted, setIsSubmitted] = useState(false); 
+    const profile = process.env.NEXT_PUBLIC_PROFILE || '';
```

Comment:
it will be reinialise with each rerender

Reply:
fixed